readme.md

```diff
@@ -47,11 +44,11 @@ for the code editor themes to work.
 
 ## Skin Installation
 
-In the zip file go to the folder **skins** and pick a skin you want to use.
+Navigate to the **skins** folder and pick a skin you want to use.
```

Comment:
Onduidelijke Instructie

Reply:
Ik snap niet hoe het onduidelijk is. Er is 1 **skins** folder en er wordt gezegd dat er naar die folder genavigeerd moet worden.